app/controllers/email/VerifyEmailController.scala

```diff
@@ -116,8 +115,6 @@ class VerifyEmailController @Inject()(val emailVerificationService: EmailVerific
     }
   }
 
-
-
   def updateContactPrefEmail(): Action[AnyContent] = (contactPreferencePredicate andThen paperPrefPredicate).async {
     implicit user =>
 
```

Comment:
There's currently a bug in this controller where you can't get to the confirmation page because you don't have the `emailChangeSuccessful` value in session. You instead get redirected to the start of the email-only journey. The redirect on line 141 needs to add the following session item: `emailChangeSuccessful -> "true"`.